README.md

```diff
@@ -42,13 +44,9 @@ npm run build
 ##### Developing:
 Run each of these commands in separate terminals:
 
-To rebuild your js every time you save:
-
+To run a development build of the project at https://localhost:3456
```

Comment:
It seems like this still says `https` while it's being served at `http`. I think the original text was fine, since the command does log the address into the console.

Reply:
Fixed ab09265ec93da3df972c1e4a00e1a26a195ac984